wolf/flows/couplings/transform.py

```diff
@@ -55,7 +55,8 @@ def __init__(self, dim, alpha):
     @overrides
     def calc_params(self, params):
         mu, log_scale = params.chunk(2, dim=self.dim)
-        scale = log_scale.mul_(0.5).tanh_().mul(self.alpha).add(1.0)
+        # scale = log_scale.mul_(0.5).tanh_().mul(self.alpha).add(1.0)
+        scale = log_scale.clone().mul_(0.5).tanh_().mul(self.alpha).add(1.0)
```

Comment:
I did not suffer the error before. Perhaps it happens for PyTorch 1.7.

I used in-place operations here to save memory. If it causes this error, we can change it to
cale = log_scale.mul(0.5).tanh().mul(self.alpha).add(1.0)